generators/client/templates/angular/src/main/webapp/app/shared/util/moment-to-string-pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Moment } from 'moment';
+import moment = require('moment');
+
+@Pipe({name: 'moment'})
+export class MomentToStringPipe implements PipeTransform {
```

Comment:
Why not use a genetic lib like this instead https://github.com/urish/angular2-moment, i font want to maintain something unless thats the only option or if the lib is too heavy for our usecase

Reply:
Sure!, will do it!